Vagrantfile

```diff
@@ -52,10 +52,13 @@ Vagrant.configure("2") do |config|
       fi
     SHELL
 
-    # Run playbook to setup /etc/hosts on target host
+    # Run vagrant specific playbook for preliminary setup
     host.vm.provision "ansible_local" do |ansible|
-      ansible.playbook = "configure_hosts.yml"
+      ansible.playbook = "configure_vagrant.yml"
```

Comment:
it seems this is a new file. Is `configure_hosts.yml` still used? I can't find any other usage with a quick GitHub search/

Reply:
The original instructions before vagrant used this file to configure the /etc/hosts file. I imagine we would still need this file if we were to run the playbook outside of vagrant.
https://github.com/openstax/cnx-deploy/pull/1342/files#diff-ba2b72f148fb98bb8e8549f4336d2893R18-R23